orttraining/orttraining/test/gradient/gradient_op_test_utils.h

```diff
@@ -7,6 +7,9 @@
 
 namespace onnxruntime {
 namespace test {
+using TestDataVector = std::tuple<std::vector<std::vector<TensorInfo>>, // Input data
```

Comment:
I would recommend std::vector<std::tuple<x,y,attr>>, so each test case can be view together. 

But it's your call.